client/src/pages/AddCaloriesPage.js

```
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { useHttp } from '../hooks/http.hook';
+import { Loader } from '../components/Loader';
+import { ProductsList } from '../components/ProductsList';
+import { TrainingList } from '../components/TrainingList';
+import { getItems } from '../http/CategoriesService';
+
+export const AddCaloriesPage = ({ token }) => {
+  const [ products, setProducts ] = useState([]);
+  const [ training, setTraining ] = useState([]);
+  const { loading } = useHttp();
+
+  const getCategories = async () => {
+    try {
+      const { data } = await getItems(token);
+      setProducts(data.products);
+      setTraining(data.training);
+    } catch (e) {}
```

why we need empty catch ? maybe we can just remove it ?

---

fixed